RubberduckTests/Refactoring/EncapsulateField/EncapsulateFieldTests.cs

```diff
@@ -21,6 +21,8 @@ public class EncapsulateFieldTests : InteractiveRefactoringTestBase<IEncapsulate
     {
         private EncapsulateFieldTestSupport Support { get; } = new EncapsulateFieldTestSupport();
 
+        private static Func<string, string> Param = EncapsulateFieldTestSupport.ParamNameBuilder;
```

Comment:
I would prefer something like `RhsParameterNameBuilder`, but it seems good-enough here in the tests.